.github/workflows/release-azure-pipelines.yml

```diff
@@ -59,6 +62,9 @@ extends:
         name: Azure-Pipelines-1ESPT-ExDShared
         image: windows-latest
         os: windows
+      # This prevents auto-injected Roslyn task from running the build again.
+      roslyn:
+        copyLogsOnly: true

```

Comment:
So what is the default behavior here if we don't specify this config? Is there a separate build job that gets invoked?

If so, I wonder if that is preferable to having an extra "build" step that runs in sequence, as jobs run in parallel.

Reply:
It would still search for a build task but not use its logs. So, it will have its own build along with build that we do separately. We could have a parallel build or something similar, but I am not sure if there are any benefits to doing that.

Reply:
One that I can think of is if it's run in a separate job, that it **might** be faster than adding a build step as part of the existing job. But that sort of depends on how long it takes to restore dependencies in that separate job. And whether or not calling "dotnet package" after the new "dotnet build" would re-build the whole project or simply re-use the build cache from dotnet build.

Reply:
That's an interesting point. Doing a bit of napkin math reveals that, on average 24 mins run of this pipeline, it adds about ~1.5% of overhead (25 seconds for this step). Install dependencies is about 40s on average.

Reply:
Interesting! That does seem fairly minor. Is install dependencies + build in a separate job that runs in parallel (Roslyn's default behaviour) faster than the added build step that runs in sequence (25 seconds)?

Reply:
Or in other words: is the default 1ES PT behaviour faster, even if it's technically doing more work? (assuming it runs in a separate job)

Reply:
Also, without that config, Roslyn injected builds take 35-40s on average compared to 15s with that config. So, it seems we are not (if at all) impacting the overall run time by anything more than 0.5%.

Reply:
Ah got it, if it it's also injecting a sequential step and that step takes longer, this definitely seems like the right way forward :) Thank you for digging into this!